lib/puppet/gettext/config.rb

```diff
@@ -48,44 +87,61 @@ def self.translation_mode(conf_path)
     end
   end
 
+  # @api private
   # Prevent future gettext initializations
   def self.disable_gettext
     @gettext_disabled = true
   end
 
-  # Attempt to initialize the gettext-setup gem
-  # @param path [String] to gettext config file
+  # @api private
```

Comment:
Private is interesting here. Not sure what the convention is. I'm ok if this says people shouldn't rely on it, but we use it outside the Module (which violates one definition of private).

Reply:
From what I understood of @branan 's comment, this was a docs only annotation, to encourage people not to rely on it. I was not intending to make a public/private distinction with anything in this module.

Reply:
Works for me.